src/libslic3r/PlaceholderParser.cpp

```diff
@@ -884,7 +884,8 @@ namespace client
                 case coPercents: output.set_d(static_cast<const ConfigOptionPercents*>(opt.opt)->values[idx]); break;
                 case coPoints:   output.set_s(to_string(static_cast<const ConfigOptionPoints  *>(opt.opt)->values[idx])); break;
                 case coBools:    output.set_b(static_cast<const ConfigOptionBools   *>(opt.opt)->values[idx] != 0); break;
-                //case coEnums:    output.set_s(opt.opt->vserialize()[idx]); break;
+                // BBS
```

Comment:
Cool, thanks for troubleshooting the regression. 
To make our life easier(easy for maintenance) in the future, could you change comment to ‘// Orca: support enum value type, treat it as vector type’ ?

Reply:
Done